internal/support/notifications/rest_subscription.go

```diff
@@ -284,6 +310,29 @@ func splitVars(vars string) []string {
 	return strings.Split(vars, ",")
 }
 
+func validateEmailAddresses(s models.Subscription) error {
+	var invalidAddrs []string
+	for _, c := range s.Channels {
+		if c.Type == models.ChannelType(models.Email) {
+			for _, m := range c.MailAddresses {
+				if strings.ContainsAny(m, "\n\r") {
+					invalidAddrs = append(invalidAddrs, m)
+				}
+			}
+		}
+	}
+	if len(invalidAddrs) > 0 {
+		resp := "Subscription " + s.Slug + " mail addresses contain CRLF: ["
```

Comment:
If my previous comment is valid then we can eliminate this in favor of the `ErrInvalidEmailAddresses` type handling the error message formatting.

Reply:
Yes, happy to remove. Done.